doc/tutorial/basic-server-installation.rst

```diff
@@ -1,58 +1,40 @@
 Basic server installation
-*************************
+=========================
 
-This chapter provides an overview of how to install Ubuntu Server Edition. You
-can also refer to this guide on
-:ref:`how to operate the installer <operate-server-installer>` for more
-information on using the installer, and to this
-:doc:`screen-by-screen reference guide <screen-by-screen>` for more
-information about each of the installer screens.
+This chapter provides an overview of how to install Ubuntu Server Edition. See also the guide on :ref:`how to operate the installer <operate-server-installer>` for more information on using the installer, and the :doc:`screen-by-screen guide <screen-by-screen>` for more information about each of the installer screens.
 
 Preparing to install
-====================
+--------------------
 
-This section explains various aspects to consider before starting the
-installation.
+This section explains various aspects to consider before starting the installation.
 
 System requirements
--------------------
+~~~~~~~~~~~~~~~~~~~
 
-Ubuntu Server Edition provides a common, minimalist base for a variety of
-server applications, such as file or print services, web hosting, email hosting,
-etc. This version supports four 64-bit architectures:
+Ubuntu Server Edition provides a common, minimalist base for a variety of server applications, such as file or print services, web hosting, email hosting, etc. This version supports four 64-bit architectures:
 
 * amd64 (Intel/AMD 64-bit)
 * arm64 (64-bit ARM)
 * ppc64el (POWER8 and POWER9)
 * s390x (IBM Z and LinuxONE)
 
-The recommended system requirements are:
+Recommended system requirements:
 
-* CPU: 1 gigahertz or better
-* RAM: 1 gigabyte or more
-* Disk: a minimum of 2.5 gigabytes
+* CPU: 1 GHz or better
+* RAM: 1 GB or more
```

Comment:
Errata welcome :)

Reply:
Anecdote: in at least my QEMU tests, 2GB was not enough to run the server installer.

Reply:
Hm it definitely did work in 1GiB back in the mists of time. It would be sort of interesting to explore what the minimal amount is now and maybe see where it runs out when trying with less than that, although I also don't think this is a huge priority (I always used to say subiquity is not designed for installing toasters!)